staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go

```diff
@@ -33,7 +33,8 @@ import (
 // The value for timeout should effectively be "forever." Obviously we don't want our tests to truly lock up forever, but 30s
 // is long enough that it is effectively forever for the things that can slow down a run on a heavily contended machine
 // (GC, seeks, etc), but not so long as to make a developer ctrl-c a test run if they do happen to break that test.
-var ForeverTestTimeout = time.Second * 30
+// Since we use one etcd lease to handle events within 60s, we use 30+60=90s in the test for forever timeout.
+var ForeverTestTimeout = time.Second * 90
```

Comment:
Why change this to 90s?

Reply:
I added comments for this "Since we use one etcd lease to handle events within 60s, we use 30+60=90s in the test for forever timeout."

Reply:
Can you give an example which test would need this long?

Reply:
@ccding you should change the const defined (also change it to a var) in store.go to make the lease shorter for testing instead of changing this global thing.